internal/distributed/querycoord/service.go

```diff
@@ -123,10 +130,21 @@ func (s *Server) init() error {
 	}
 	s.etcdCli = etcdCli
 	s.SetEtcdClient(etcdCli)
-	s.queryCoord.SetAddress(Params.GetAddress())
+	s.queryCoord.SetAddress(rpcParams.GetAddress())
+
+	if params.MetaStoreCfg.MetaStoreType.GetValue() == util.MetaStoreTypeTiKV {
+		log.Info("Connecting to tikv metadata storage.")
+		s.tikvCli, err = getTiKVClient(&paramtable.Get().TiKVCfg)
+		if err != nil {
+			log.Debug("QueryCoord failed to connect to tikv", zap.Error(err))
```

Comment:
check the log level.
Should be warn here

Reply:
Done. Likely from copy-paste.
https://github.com/milvus-io/milvus/issues/26831 to fix the existing ones.